G1ANT.Addon.Net/G1ANT.Addon.Net/Variables/TimeoutConnectVariable.cs

```diff
@@ -5,7 +5,7 @@ namespace G1ANT.Addon.Net.Commands
 {
     [Variable(
         Name = "timeoutconnect",
-        Tooltip = "Determines the timeout value (in ms) for the [is.accessible](G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/IsAccessibleCommand.md) and [ping](G1ANT.Addon/G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/PingCommand.md) commands; the default value is 1000 (1 second).")]
+        Tooltip = "Determines the timeout value (in ms) for the is.accessible and ping commands; the default value is 1000 (1 second)")]
```

Comment:
Determines the timeout value (in ms) for the `is.accessible` and the `ping` commands; the default value is 1000 (1 second)